mne/io/reference.py

```diff
@@ -97,11 +128,16 @@ def _apply_reference(inst, ref_from, ref_to=None):
                 'using the apply_proj() method function before the desired '
                 'reference can be set.'
             )
+
     for i in projs_to_remove:
         del inst.info['projs'][i]
```

Comment:
@wmvanvliet I tried to replace this with a clean `inst.del_proj(projs_to_remove)` but it would not work as I cannot remove an EEG avg ref if it's already applied :(

@wmvanvliet is it common to have an avg ref on a subset of channels and some in bipolar? I find weird that with the current code I can have an average ref, then change some channels the remaining EEG channels are not zero mean anymore.

A clean way would be to remove all the channels that are not part of the bipolar montage but I don't know if it's acceptable.

thoughts?